doc/source/wsrep-system-index.rst

```diff
@@ -1238,5 +1275,72 @@ is determined by bitmask:
 .. seealso:: `MySQL wsrep option: wsrep_sync_wait
              <https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-sync-wait>`_
 
+.. variable:: wsrep_trx_fragment_size
+
+   :cli: ``--wsrep-trx-fragment-size``
+   :conf: Yes
+   :scope: Global, Session
+   :dyn: Yes
+   :default: 0
+
+Defines the the streaming replication fragment size. This variable is measured in the units defined by ``wsrep_trx_fragment_unit``. The minimum value is 0 and the maximum value is 2147483647.
+
+As of *Percona Server for MySQL* 8.0.27, you can update the variable with a `set_var hint <https://dev.mysql.com/doc/refman/8.0/en/optimizer-hints.html#optimizer-hints-set-var>`__.
+
+.. code-block:: mysql
+
+   mysql> SELECT @@@wsrep_trx_fragment_units; SELECT @@wsrep_trx_fragment_size; 
+   +------------------------------+
+   | @@wsrep_trx_fragment_unit    |
+   +==============================+
+   | statements                   |
+   +------------------------------+
+   | @@wsrep_trx_fragment_size    |
+   +------------------------------+
+   | 3                            |
+   +------------------------------+
+   mysql> SELECT /*+ SET_VAR(wsrep_trx_fragment_size=5) */ @@wsrep_trx_fragment_size;
```

Comment:
I think it is also better to tell users that one can use SET_VAR() extension for DMLs as well and can be useful for streaming large statement within a transaction.

```
node1> begin;
Query OK, 0 rows affected (0.00 sec)

node1> INSERT /*+SET_VAR(wsrep_trx_fragment_size = 100)*/ INTO t1 SELECT * FROM t1;
Query OK, 65536 rows affected (15.15 sec)
Records: 65536  Duplicates: 0  Warnings: 0

node1> UPDATE /*+SET_VAR(wsrep_trx_fragment_size = 100)*/ t1 SET i=2;
Query OK, 131072 rows affected (1 min 35.93 sec)
Rows matched: 131072  Changed: 131072  Warnings: 0

node1> DELETE /*+SET_VAR(wsrep_trx_fragment_size = 10000)*/ FROM t1;
Query OK, 131072 rows affected (15.09 sec)



Meanwhile on node2, while UPDATE is running on node1

node2> set session transaction_isolation = 'READ-UNCOMMITTED';
Query OK, 0 rows affected (0.00 sec)

node2> SELECT * FROM t1 LIMIT 10;
+------+
| i    |
+------+
|    2 |
|    2 |
|    2 |
|    2 |
|    2 |
|    2 |
|    2 |
|    2 |
|    2 |
|    2 |
+------+
10 rows in set (0.00 sec)
```

Reply:
Updated examples